sdk/core/core-amqp/src/errors.ts

```diff
@@ -676,7 +677,7 @@ export function translate(err: unknown): MessagingError | Error {
     return error;
   }
 
-  if (err instanceof Error && err.name === "MessagingError") {
+  if (isError(err) && err.name === "MessagingError") {
     // already translated
     return err as MessagingError;
```

Comment:
Should be unnecessary now.

```suggestion
    return err;
```